komorebi/src/static_config.rs

```diff
@@ -348,6 +358,12 @@ impl From<&WindowManager> for StaticConfig {
             }
         }
 
+        let border_colours = ActiveWindowBorderColours {
```

Comment:
Let's check if the value of `BORDER_COLOUR_SINGLE` is `0` here (https://github.com/LGUG2Z/komorebi/blob/master/komorebi/src/main.rs#L177) and set `border_colours` to `None` if it is, and then immediately wrap `border_colours` as a `Some(ActiveWindowBorderColors{...}` if it has actually been set, because if people set the equivalent of `0` (no colour) in their static configs, when/if they enable the active window border in the future, it won't be visible.

Reply:
Or alternatively, if it hasn't been set, maybe we can just set the colours from the example config? 🤔